src/api/features/features.api.js

```diff
@@ -529,108 +525,161 @@ export const identify = (config) => {
+ * @param {CoordinateSystem} outputProjection In which projection the feature should be in.
+ * @param {Object} [options]
+ * @param {String} [options.lang] The lang the title of the feature should be look up. Some features
+ *   do provide a title per lang, instead of a all-purpose title. In this case we need the lang ISO
+ *   code to be able to decide which title the feature will have. Default is `en`
+ * @returns {LayerFeature}
+ */
+function parseGeomAdminFeature(
+    layer,
+    featureMetadata,
+    featureHtmlPopup,
+    outputProjection,
+    options = {}
+) {
+    const { lang = 'en' } = options
+    const featureGeoJSONGeometry = featureMetadata.geometry
+    let featureExtent = []
+    if (featureMetadata.bbox) {
+        featureExtent.push(...featureMetadata.bbox)
+    }
+    let featureName = featureMetadata.id
+    if (featureMetadata.properties) {
+        const { name = null, title = null, label = null } = featureMetadata.properties
+        const titleInCurrentLang = featureMetadata.properties[`title_${lang}`]
+        if (label) {
+            featureName = label
+        } else if (name) {
+            featureName = name
+        } else if (title) {
+            featureName = title
+        } else if (titleInCurrentLang) {
+            featureName = titleInCurrentLang
+        }
+    }
+
+    if (outputProjection.epsg !== LV95.epsg) {
+        if (featureExtent.length === 4) {
+            featureExtent = projExtent(LV95, outputProjection, featureExtent)
+        }
+    }
+
+    return new LayerFeature({
+        layer,
+        id: featureMetadata.id,
+        name: featureName,
+        data: featureHtmlPopup,
+        coordinates: getGeoJsonFeatureCoordinates(featureGeoJSONGeometry, LV95, outputProjection),
+        extent: featureExtent,
+        geometry: featureGeoJSONGeometry,
+    })
+}
+
 /**
  * Loads a feature metadata and tooltip content from this two endpoint of the backend
  *
- * - http://api3.geo.admin.ch/services/sdiservices.html#identify-features
+ * - https://api3.geo.admin.ch/services/sdiservices.html#feature-resource
  * - http://api3.geo.admin.ch/services/sdiservices.html#htmlpopup-resource
  *
  * @param {GeoAdminLayer} layer The layer from which the feature is part of
- * @param {String | Number} featureID The feature ID in the BGDI
+ * @param {String | Number} featureId The feature ID in the BGDI
  * @param {CoordinateSystem} outputProjection Projection in which the coordinates (and possible
  *   extent) of the features should be expressed
- * @param {String} lang The language for the HTML popup
+ * @param {String} lang The language for the HTML popup. Default is `en`.
  * @returns {Promise<LayerFeature>}
  */
-const getFeature = (layer, featureID, outputProjection, options = {}) => {
-    const { lang = 'en', coordinates = null, imageDisplay = null, mapExtent = null } = options
+const getFeature = (layer, featureId, outputProjection, lang = 'en') => {
```

Comment:
In which case the `getFeature` is used, I mean in which case we need to get the htmlpopup and the feature info at the same time ?

Reply:
at app startup when a pre-selected feature is present. Only use case I see

Reply:
I hesitated to move the loading of the HTML popup directly in the FeatureDetail component, and only deal with metadata in this file...
But then we have go-live in 2 (work) days and didn't want to spend too much time here...

Reply:
there's a second usage of this function in the feature store, when a features are updated after a lang switch